TW 743 Improve governance and security introduction (#4356)
postmanlabs/postman-docs: src/pages/docs/api-governance/api-governance-overview.md

    url: "https://blog.postman.com/iterative-api-governance-strategy-postman/"
---

> API governance is the application of rules that promote consistent API behaviors across your organization's API landscape. API security is the application of rules that promote security-first API behaviors. The _Postman API Governance_ and _Postman API Security_ features can identify inconsistencies or weaknesses in your APIs, then recommend possible fixes or improvements. These features can enable you to create better and more secure APIs that follow industry best practices.

Reviewer: I'd rewrite around the word "application" in the first and second sentence, just because app/application is such an overloaded term.

Author: Good call, thanks!

> By automatically applying Postman API Governance and API Security rules at the beginning of API development, you can automate your governance and security review processes instead of relying on manual reviews. Consistently enforcing Postman API Governance and API Security standards enables you to develop high-quality APIs and streamlines collaboration across all parts of the API development lifecycle.  

Reviewer: A little nit: do you _apply_ rules? That makes it sound like you're doing something, like making changes to the API. You're really maybe checking rules?

    url: "https://blog.postman.com/iterative-api-governance-strategy-postman/"
---

> API governance is the practice of defining and applying development rules that promote consistent API behaviors across your organization's API landscape. And a robust API security posture means that your organization has development rules that promote security-first API behaviors. The _Postman API Governance_ and _Postman API Security_ features can identify inconsistencies or weaknesses in your APIs, then recommend possible fixes or improvements. These features can enable you to create better and more secure APIs that follow industry best practices.

Author: ```suggestion
API governance is the practice of defining and applying development rules that promote consistent API behaviors across your organization's API landscape. And a robust API security posture means that your organization has development rules that promote security-first API behaviors. The _Postman API Governance_ and _Postman API Security_ features can identify inconsistencies or weaknesses in your APIs, then recommend possible fixes or improvements. With these features, you can create better and more secure APIs that follow industry best practices.
```

Postman automatically checks your API definition and requests using predefined API Governance and API Security rules.

> [Enterprise teams](https://www.postman.com/pricing/) can also create custom rules. See the [Configurable rules overview](/docs/api-governance/configurable-rules/configurable-rules-overview/) for more details.

By checking your API at the beginning of the development process, you can automate your organization's governance and security review processes instead of relying on manual reviews. Consistently enforcing Postman API Governance and API Security standards enables you to develop high-quality APIs and streamlines collaboration across all parts of the API development lifecycle.

Postman applies API Governance and API Security rules at the following phases of API development:

* **API definition** - Postman applies API Governance and API Security rules to your [API definition](/docs/api-governance/api-definition/api-definition-warnings/) and shows rule violations that might impact your definition's governance and security postures.

> * **Requests** - Postman applies API Security rules configured for your [API requests](/docs/api-governance/api-testing/api-testing-warnings/) when you send requests to any API using either the Postman web app or the Postman desktop app.

Reviewer 2: @HeatherStenson lets discuss a way to link this to the API producer and API consumer lifecycle that we use in almost every all-hands. 
This need not be part of the existing PR, but something to start thinking about - as we foresee the API Governance and API Security products diverging a bit as we move ahead.

Author: Thanks @savageCoder55! I'll make a Jira for this and we can start discussing specifics.
